Vagrantfile

```diff
@@ -0,0 +1,17 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+Vagrant.configure('2') do |config|
+  config.vm.box = 'ubuntu/trusty64'
```

Comment:
Good catch @feanil . I originally created this by converting a `Dockerfile` into the `Vagrantfile`, using Trusty for the Docker support in the 14.04 kernel.

`Vagrantfile` has been updated.